src/systems/hubs-systems.js

```diff
@@ -6,9 +6,28 @@ import { HoverMenuSystem } from "./hover-menu-system";
 import { SuperSpawnerSystem } from "./super-spawner-system";
 import { HapticFeedbackSystem } from "./haptic-feedback-system";
 import { SoundEffectsSystem } from "./sound-effects-system";
+import { proxiedUrlFor } from "../utils/media-utils";
+import { load, prepareForRender } from "../utils/preload";
+import cameraModelSrc from "../assets/camera_tool.glb";
+
+async function preloadPenAndCamera(sceneEl) {
+  const objects = [];
+
+  const pen = await load(
+    proxiedUrlFor("https://asset-bundles-prod.reticulum.io/interactables/DrawingPen/DrawingPen-34fb4aee27.gltf")
```

Comment:
If you trace the crazy path by which pen gets loaded, you will see that we don't need to proxy this.

Reply:
I don't need to _resolve_ this but do still want to proxy it